### PR TITLE
Update actions

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8654

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md [hunk 1]
```diff
@@ -121,6 +122,10 @@ These actions are applied to files in the `SHA1` or the `InitiatingProcessSHA1`
 - **Allow/Block**—automatically adds the file to your [custom indicator list](manage-indicators.md) so that it is always allowed to run or blocked from running. You can set the scope of this action so that it is taken only on selected device groups. This scope is independent of the scope of the rule.
 - **Quarantine file**—deletes the file from its current location and places a copy in quarantine
 
+### Actions on users
+
+- **Mark user as compromised**-sets the users risk level to "high" in Azure Active Directory, triggering corresponding [identity protection policies](https://docs.microsoft.com/azure/active-directory/identity-protection/overview-identity-protection#risk-levels).
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md:127]: Based on the context that "risk level" is a property that belongs to the user, a possessive form of the noun "user" is required. Seeing that the action name refers to a singular user, I believe the possessive is "user's".
```suggestion
- **Mark user as compromised**-sets the user's risk level to "high" in Azure Active Directory, triggering the corresponding [identity protection policies](https://docs.microsoft.com/azure/active-directory/identity-protection/overview-identity-protection#risk-levels).
```
Also added denominator "the", in reference to [identity protection policies].

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md [hunk 2]
```diff
@@ -113,6 +113,7 @@ These actions are applied to devices in the `DeviceId` column of the query resul
 - **Collect investigation package**—collects device information in a ZIP file. [Learn more about the investigation package](respond-machine-alerts.md#collect-investigation-package-from-devices)
 - **Run antivirus scan**—performs a full Microsoft Defender Antivirus scan on the device
 - **Initiate investigation**—starts an [automated investigation](automated-investigations.md) on the device
+- **Restrict app execution**—sets restrictions on device to allow only files that are signed with a Microsoft-issued certificate to run. [Learn more about restricting app execution](respond-machine-alerts.md#restrict-app-execution)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md:116]:
```suggestion
- **Restrict app execution**—sets restrictions on the device to allow only files that are signed with a Microsoft-issued certificate to run. [Learn more about restricting app execution](respond-machine-alerts.md#restrict-app-execution)
```

File: windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md [hunk 3]
```diff
@@ -121,6 +122,10 @@ These actions are applied to files in the `SHA1` or the `InitiatingProcessSHA1`
 - **Allow/Block**—automatically adds the file to your [custom indicator list](manage-indicators.md) so that it is always allowed to run or blocked from running. You can set the scope of this action so that it is taken only on selected device groups. This scope is independent of the scope of the rule.
 - **Quarantine file**—deletes the file from its current location and places a copy in quarantine
 
+### Actions on users
+
+- **Mark user as compromised**-sets the user's risk level to "high" in Azure Active Directory, triggering the corresponding [identity protection policies](https://docs.microsoft.com/azure/active-directory/identity-protection/overview-identity-protection#risk-levels).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md:127]:
```suggestion
- **Mark user as compromised**—sets the user's risk level to "high" in Azure Active Directory, triggering the corresponding [identity protection policies](https://docs.microsoft.com/azure/active-directory/identity-protection/overview-identity-protection#risk-levels).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md (hunk 3 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md]: Good catch. I did not notice that detail.

File: windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md (hunk 2 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/custom-detection-rules.md]: Fair point. I should have noticed, but somehow I didn't. Good catch.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Pull request merged**